test/autowaiting.jest.js

```diff
@@ -210,7 +208,7 @@ describe('Auto waiting should not hang when', () => {
         setTimeout(() => window.stop(), 100);
       }, server.EMPTY_PAGE);
   });
-  it('calling window.stop sync', async({page, server, httpsServer}) => {
+  it('calling window.stop', async({page, server, httpsServer}) => {
```

Comment:
Why didn't you like sync? The point was that is was stopping synchronously with the navigation...

Reply:
Ops, I read “async” as the test above. I’ll undo this change :)